src/components/GradesView/GradebookTable/Fields.jsx

```diff
@@ -29,16 +29,16 @@ Username.propTypes = {
 };
 
 /**
- * Fields.Email
- * Simple label field for email value.
- * @param {string} email - email for display
+ * Fields.SimpleText
+ * Simple label field for text value.
+ * @param {string} value - value for display
  */
-const Email = ({ email }) => <span className="wrap-text-in-cell">{email}</span>;
-Email.propTypes = {
-  email: PropTypes.string.isRequired,
+const SimpleText = ({ value }) => (<span className="wrap-text-in-cell">{value}</span>);
+SimpleText.propTypes = {
```

Comment:
Nit: is there a reason to call it `SimpleText` and not just `Text`? Not a problem but is more verbose.